src/Illuminate/Database/Eloquent/Factories/Factory.php

```diff
@@ -626,10 +627,13 @@ public function modelName()
     {
         $resolver = static::$modelNameResolver ?: function (self $factory) {
             $factoryBasename = Str::replaceLast('Factory', '', class_basename($factory));
+            $rootNamespace = Container::getInstance()
+                            ->make(Application::class)
+                            ->getNamespace();
```

Comment:
What if this is called from outside of the Laravel framework?

Reply:
As in a Laravel app the `Container` is implemented by `Application` would this make sense?

~~~php
$rootNamespace = method_exists(Container::getInstance(), 'getNamespace')
    ? Container::getInstance()->getNamespace()
    : 'App\\';
~~~

If it makes sense it could be extracted to a method as it would be needed in more places.

Reply:
Didn't see you commit https://github.com/laravel/framework/commit/8bf80de4b96b507fcc74a3e217d834882fefa67a

Guess that is a better solution